doc/themes/the-things-stack/assets/css/theme.scss

```diff
@@ -217,4 +217,8 @@ html.has-navbar-fixed-top {
   }
 }
 
+details.with-padding {
+  padding-bottom: 1em;
+}
```

Comment:
OK so what is now the rule for using this?

Is it when there is _no_ heading after the details?

Shouldn't we always add padding after the details instead?

The problem I have with this, is that content writers now have to think about styling. That goes against the concept of using Markdown as a documentation tool; separate designers from domain experts.

Reply:
Switched to using `details + p`  and `details + blockquote` selectors instead. This removes the need from content writers to think about styling, and covers our case.

We can also simply add padding after every `<details>` section, but that pushes the following headers too far down and makes the doc look kinda bad.